lego/apps/articles/migrations/0012_auto_20190304_1952.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 2.1.7 on 2019-03-04 19:52
```

Comment:
You can merge all these migrations into one (0009 - 0012)

Reply:
How do I merge the migrations? 💯